src/components/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import { clsx } from 'clsx';
+import React, { ReactComponentElement, ReactElement, ReactNode } from 'react';
+
+
+import styles from './Tabs.module.scss';
+
+type TabsItem = {
+  title: string;
+  url: string;
+  name: string;
+  disabled?: boolean
+};
+
+export type TabsProp = {
+  items: TabsItem[];
+  activeTab: string | string[];
+  counts?: any;
+  onClick: (url: string) => void;
```

Comment:
On click should be a url or a function what be better?

Reply:
Actually like this the onclick is going to be one and the same for every link. I guess this is not correct, right?

Reply:
yes....

Reply:
are there use cases where the use is different?

Reply:
Ok for now we can leave it like this. But in the future we might need to pass the onClick function instead of just the URL

Reply:
we are passing an onClick function to the tab
But the onclick function passes a string as an argument

```jsx
<Tabs 
	activeTab={activeTab} 
	onClick={(value: string) => {
		// clicked tab value
    	console.log(value)
  	}} 
/>